includes/Mapper/class-contactmapper.php

```diff
@@ -0,0 +1,190 @@
+<?php
+/**
+ * Contact mapper
+ *
+ * @package OmnisendLifterLMSPlugin
+ */
+
+declare(strict_types=1);
+
+namespace Omnisend\LifterLMSAddon\Mapper;
+
+use Omnisend\SDK\V1\Contact;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class ContactMapper
+ */
+class ContactMapper {
+	private const CUSTOM_PREFIX  = 'lifter_lms';
+	private const CONSENT_PREFIX = 'lifter_lms';
+
+	/**LifterLMSAddon
+	 * Get Contact object
+	 *
+	 * @param array  $mapped_fields
+	 *
+	 * @return Contact object
+	 */
+	public function get_omnisend_contact( array $mapped_fields ): Contact {
+		$options = get_option( 'omnisend_lifterlms_options' );
+		$contact = new Contact();
+
+		$contact->set_email( $mapped_fields['email_address'] );
+		$contact->set_phone( $mapped_fields['llms_phone'] );
+		$contact->set_first_name( $mapped_fields['first_name'] ?? '' );
+		$contact->set_last_name( $mapped_fields['last_name'] ?? '' );
+		$contact->set_postal_code( $mapped_fields['llms_billing_zip'] ?? '' );
+		$contact->set_address( $mapped_fields['llms_billing_address_1'] . ' ' . $mapped_fields['llms_billing_address_2'] ?? '' );
+		$contact->set_state( $mapped_fields['llms_billing_state'] ?? '' );
+		$contact->set_country( $mapped_fields['llms_billing_country'] ?? '' );
+		$contact->set_city( $mapped_fields['llms_billing_city'] ?? '' );
+
+		$contact->set_welcome_email( true );
+
+		if ( ( isset( $mapped_fields['llmsconsentEmail'] ) && isset( $mapped_fields['email_address'] ) ) || ! isset( $options['filter_lms_consent_setting'] ) ) {
+			$contact->set_email_consent( self::CONSENT_PREFIX );
+			$contact->set_email_opt_in( $mapped_fields['email_address'] );
```

Comment:
email opt int shouldn't be the email

Reply:
corrected both email_opt_in and phone_opt_in